.github

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [Optional ] Please commit should be in the present tense 

![image](https://user-images.githubusercontent.com/74744787/164030940-3668aaf2-a562-49be-aab0-4ddcab2ae005.png)